### PR TITLE
Removes icon-on / icon-off from Smartfridge Types

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
Please review the replacetext, only reason I have that is for admin shitlery but I can remove it if we want to BTFO admins from changing icons of fridges.

No actual sprite changes, I just sync'd iconstates to fit.

Purpose: To remove 2 vars.

### CODE REVIEW

File: code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm [hunk 1]
```diff
@@ -40,10 +38,13 @@
 	update_icon()
 
 /obj/machinery/smartfridge/update_icon()
+	var/startstate = initial(icon_state)
+	if((icon_state != startstate)&&(icon_state != "[startstate]-off"))
+		startstate = replacetext(icon_state, "-off", "") //admins ruin everything!
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm]: ^^

Comment (ninjanomnom) [reply code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm]: Don't bother with this

Comment (ninjanomnom) [inline code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm]: Just handle it normally, dont't bother with badmin checks

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
c-c-c-can you fix the labels

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**